content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: "Configuring Dynamic DNS"
+description: "Configuring Dynamic DNS"
```

Comment:
The description front matter should not just be the same as the title of the article. Maybe uses something like, "This article provides information on configuring dynamic DNS on your TrueNAS." or some variation. Try to summarize what information is in this article or what the reader will find in it.